robot/control/Inc/modules/IMUModule.hpp

```diff
@@ -7,17 +7,43 @@
 #include "LockedStruct.hpp"
 #include <memory>
 
+/**
+ * Module interfacing with IMU and handling IMU status
+ */
 class IMUModule : public GenericModule {
 public:
-    // How many times per second this module should run
-    static constexpr float kFrequency = 200.0f; // Hz
+    /**
+     * Number of times per second (frequency) that BatteryModule should run (Hz)
```

Comment:
Shouldn't this be IMUModule?

Reply:
Yes! I went through modules again and found a few errors which I'll add in a fix commit.